### PR TITLE
Disable console title workaround for >=RS4

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/25531

cc @joperezr @ianhays

### CODE REVIEW

File: src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs [hunk 1]
```diff
@@ -44,6 +44,9 @@ public static partial class PlatformDetection
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 15063;
         public static bool IsWindows10Version1709OrGreater => 
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16299;
+        // This needs to be just adjusted with the actual build number and date.
```
Comment (danmosemsft) [inline src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: I have 1709/16299 and it DOES fail (ie. does not throw)

Comment (ViktorHofer) [reply src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: That's weird. If so, shouldn't it also fail on Helix/CI? And also weird that no one else yet reported that issue.

Comment (danmosemsft) [reply src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: @MattGal what build of Windows 10 do the Helix test machines use currently?

Comment (MattGal) [reply src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: Windows.10.Amd64 : 14393.1198.amd64fre.rs1_release_sec.170427-1353
Windows.10.Amd64.ClientRS2: 15063.0.x86fre.rs2_release.170317-1834
Windows.10.Amd64.ClientRS3: 17025.1000.x86fre.rs_prerelease.171020-1626
Windows.10.Amd64.ServerRS3: 16299.15.x86fre.rs3_release.170928-1534

So all our RS3 SKUs are >= 16299.

Comment (ViktorHofer) [reply src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: Maybe the test is only failing sometime... I don't see it failing here: https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20171127.01

Comment (danmosemsft) [reply src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs]: Maybe for builds in this range just accept either success or throw?

This isn't worth thinking about too much, setting huge console titles on certain OS versions is not a mainstream scenario :)

**Comment by stephentoub** [comment]
@dotnet-bot test Tizen armel Debug Build please

**Pull request merged**